ambry-store/src/main/java/com.github.ambry.store/Log.java

```diff
@@ -119,8 +119,7 @@ public void appendFrom(ReadableByteChannel channel, long size)
         bytesWritten);
   }
 
-  @Override
-  public void writeFrom(ReadableByteChannel channel, long offset, long size)
+  void writeFrom(ReadableByteChannel channel, long offset, long size)
```

Comment:
Done. This is going to be removed in the next PR so that documentation is minimal.